File: Santander/Santander/Models/Entidades/Solicitudes/SolicitudTransferencia.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Santander.Models.Entidades.Solicitudes
{
    public class SolicitudTransferencia
    {
        public String TarjetaOrigen;

        public String TarjetaDestino;

        public double Monto;

        public bool PagoTerceros;

        public String Detalle;
    }
}

File: DummySantander/app/src/main/java/com/ds/dummysantander/SolicitudTransferencia.java
package com.ds.dummysantander;

public class SolicitudTransferencia {
    private String TarjetaOrigen;
    private String TarjetaDestino;
    private double Monto;
    private boolean PagoTerceros;
    private String Detalle;

    public SolicitudTransferencia(String tarjetaOrigen, String tarjetaDestino, double monto, boolean pagoTerceros, String detalle) {
        super();
        this.TarjetaOrigen = tarjetaOrigen;
        this.TarjetaDestino = tarjetaDestino;
        this.Monto = monto;
        this.PagoTerceros = pagoTerceros;
        this.Detalle = detalle;
    }

    public SolicitudTransferencia()
    {}

}
